docs/cypress/e2e/links.cy.ts

```diff
@@ -38,8 +38,8 @@ describe(`All pages on Sitemap`, () => {
       /** Check all the internal links */
       cy.get(`a[href^='/']`).each((el) => hrefOnSitemap(el, link, allLinks));
 
-      /** Check all the external links and internal links with hash */
-      cy.get(`a:not([href^='/']), a[href*='#']`).each((el) =>
```

Comment:
remove the # check from link checker. Because the # doesn't change the status code of a request, the test doesn't really check for if a # exist. A ticket is cut to follow up on the test.